packages/airnode-node/package.json

```diff
@@ -12,7 +12,7 @@
     "build": "yarn run clean && yarn run compile",
     "clean": "rimraf -rf *.tsbuildinfo ./dist *.tgz",
     "compile": "tsc --build tsconfig.build.json",
-    "dev:invoke": "ts-node src/cli/invoke.ts",
+    "dev:invoke": "ts-node src/cli/validate-config.ts && ts-node src/cli/invoke.ts",
```

Comment:
I think the validate-config script should be in a separate command and not included in `dev:invoke`. You can use invoke even with invalid config when you want to test something specific.

Reply:
I just don't like that `dev:testApi` calls `test-api.ts`, but `dev:invoke` calls both validate and invoke and we have no script for validation only.

Reply:
I understand. I still think this would be actually a less confusing approach since it will behave the same for local invocation and the container. If you don't have a hard preference I would keep it this way.

Reply:
No hard preference.